src/frontends/common/src/complex_type_mark.cpp

```diff
@@ -315,3 +320,48 @@ ov::Output<ov::Node> ComplexTypeMark::convert_like(const NodeContext& context,
 
     return context.mark_node(make_shared<v1::ConvertLike>(input, like_data));
 }
+
+ov::Output<ov::Node> ComplexTypeMark::abs(const NodeContext& context, const ov::Output<ov::Node>& data) {
+    auto data_complex = as_type_ptr<ComplexTypeMark>(data.get_node_shared_ptr());
+
+    if (data_complex) {
+        // abs of complex number sqrt(real^2 + imag^2)
+        auto data = data_complex->get_data();
+
+        // compute element-wise square for complex representation
+        auto const_two = context.mark_node(make_shared<v0::Constant>(element::i32, Shape{}, 2));
+        const_two = context.mark_node(make_shared<v1::ConvertLike>(const_two, data));
+        auto squared_data = context.mark_node(make_shared<v1::Power>(data, const_two));
+
+        // compute sum of squared real and imaginary parts
+        auto const_minus_one = context.mark_node(make_shared<v0::Constant>(element::i32, Shape{}, -1));
+        auto complex_abs = context.mark_node(make_shared<v1::ReduceSum>(squared_data, const_minus_one, false));
+
+        return context.mark_node(make_shared<v0::Sqrt>(complex_abs));
+    }
+
+    return context.mark_node(make_shared<v0::Abs>(data));
+}
+
+ov::Output<ov::Node> ComplexTypeMark::exp(const NodeContext& context, const ov::Output<ov::Node>& data) {
+    auto data_complex = as_type_ptr<ComplexTypeMark>(data.get_node_shared_ptr());
+
+    if (data_complex) {
+        auto real = data_complex->get_real();
+        auto imag = data_complex->get_imag();
+
+        // exp of complex number e^real * cos(imag) + i e^real * sin(imag)
+        auto real_exp = context.mark_node(make_shared<v0::Exp>(real));
+
+        auto imag_cos = context.mark_node(make_shared<v0::Cos>(imag));
+        auto imag_sin = context.mark_node(make_shared<v0::Sin>(imag));
+
+        auto res_real = context.mark_node(make_shared<v1::Multiply>(real_exp, imag_cos));
+        auto res_imag = context.mark_node(make_shared<v1::Multiply>(real_exp, imag_sin));
+
+        return context.mark_node(make_shared<ComplexTypeMark>(res_real, res_imag));
+        ;
+    }
+
+    return context.mark_node(make_shared<v0::Exp>(data));
+}
```

Comment:
please move it to common translators. I don't want ComplexTypeMark to be extended for each operation.
Imagine, we have several movement operations for which we also need common logic. So let us have it in common translators.

Reply:
agreed offline to do this in separate PR. 
@mvafin

Reply:
Will do in a separate PR